.eslintrc.js

```diff
@@ -17,25 +17,29 @@ module.exports = {
   rules: {
     'no-console': 'error',
     'prettier/prettier': [
-      'error', {
-        'endOfLine': 'auto'
-      }
+      'error',
+      {
+        endOfLine: 'auto',
+      },
     ],
-    // TODO remove when done killing `any`s and making tsc strict
-    '@typescript-eslint/ban-ts-comment': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-    '@typescript-eslint/no-unsafe-assignment': 'off',
-    '@typescript-eslint/no-unsafe-call': 'off',
-    '@typescript-eslint/no-unsafe-member-access': 'off',
-    '@typescript-eslint/no-unsafe-return': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'warn',
+    '@typescript-eslint/no-confusing-non-null-assertion': 'warn',
+    '@typescript-eslint/no-explicit-any': 'warn',
+    '@typescript-eslint/no-extraneous-class': 'warn',
+    '@typescript-eslint/no-implicit-any-catch': 'warn',
+    '@typescript-eslint/no-invalid-void-type': 'warn',
+    '@typescript-eslint/prefer-ts-expect-error': 'warn',
```

Comment:
Neaaaat, didn't know about `@ts-expect-error`; will use at work too 🙂

Reply:
Me neither, and the best part is it enforces your desire for there to be a REASON at the end of that comment. :)